monkey/monkey_island/cc/repository/mongo_agent_event_repository.py

```diff
@@ -26,13 +48,17 @@ def __init__(
         self._events_collection = mongo_client.monkey_island.events
         self._serializers = serializer_registry
         self._encryptor = encryptor
+        self._events_collection.create_index(EVENT_TYPE_FIELD)
```

Comment:
Can you provide docs to what this does? Doesn't indexes have to be added on push? 🤔

Reply:
```
If you want to improve your query performance, it can be helpful to create indexes for your MongoDB collections. Indexes help make query requests more efficient because they store data in a way that makes it quick and easy to traverse. With the help of an index, MongoDB can locate documents that match your query criteria without performing a full collection scan. Instead, MongoDB uses the index, which allows it to limit the number of documents it examines.
```
Got it from [here](https://kb.objectrocket.com/mongo-db/how-to-create-an-index-for-a-mongodb-collection-in-python-371)